src/nspawn/nspawn-register.c

```diff
@@ -257,6 +270,7 @@ int allocate_scope(
                 const char *machine_name,
                 pid_t pid,
                 const char *slice,
+                bool enable_fuse,
```

Comment:
So we have this rule in our codebase that if a function gains the 2nd argument of type "bool" (i.e. that indicates some sort of flag), it should get a proper flags argument. i.e. please introduce:

```c
typedef enum AllocateScopeFlags {
        ALLOCATE_SCOPE_ALLOW_PIDFD = 1 << 0,
        ALLOCATE_SCOPE_ENABLE_FUSE = 1 << 1, 
} AllocateScopeFlags;
```

And then make `AllocateScopeFlags flags` the last argument that `allocate_scope()` takes, and drop the two bool arguments.

Reply:
Done.